src/utils/Data.ts

```diff
@@ -57,14 +48,14 @@ const fetchByMicroflow = (actionName: string, contextObj: MxObject, mxform: mxui
             context,
             origin: mxform,
             callback: (mxObjects: MxObject[]) => resolve(mxObjects),
-            error: error => reject(`An error occurred while retrieving data via microflow: ${actionName}: ${error.message}`)
+            error: error => reject(new Error(`An error occurred while retrieving data via microflow: ${actionName}: ${error.message}`))
         });
     });
 };
 
 const fetchByNanoflow = (nanoflow: Data.Nanoflow, contextObj: MxObject, mxform: mxui.lib.form._FormBase, inputParameterEntity: string): Promise<MxObject[]> => {
     if (contextObj.getEntity() !== inputParameterEntity) {
-        Promise.reject("Input parameter entity does not match the context object type");
+        Promise.reject(new Error("Input parameter entity does not match the context object type"));
```

Comment:
Shouldn't this be `return Promise.reject`, or does this return as well?

Reply:
thinks it should return too.